jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -281,10 +281,9 @@ async def stage_3_process(app: Flask, qsm: QueueService):
             app.logger.debug(f'Created Involuntary Dissolution Filing with ID: {filing.id}')
             await put_filing_on_queue(filing.id, app, qsm)
 
-            batch_processing.step = BatchProcessing.BatchProcessingStep.DISSOLUTION
-            batch_processing.status = BatchProcessing.BatchProcessingStatus.COMPLETED
+            batch_processing.status = BatchProcessing.BatchProcessingStatus.QUEUED
```

Comment:
Do we want the filer to handle changing the step to `DISSOLUTION`?

Reply:
hmmmm...think we can keep it in the job. 😕 💭

Reply:
I think the filer should change the step from `QUEUED` to `DISSOLUTION`.  Otherwise, not sure how we would be able to tell the dissolution filing went through in a timely way for a business being dissolved.

My original vision was to have another listener that would process any QUEUED entries in the dissolution process but decided not to do it for the MVP implementation.

Let me know if you have more thoughts about this @chenhongjing and @leodube-aot

Reply:
Was Leo talking about changing `BatchProcessingStep` from `WARNING_LEVEL_2` to `DISSOLUTION` instead of changing `BatchProcessingStatus` from `COMPLETED` to `QUEUED`?

Reply:
I was wondering about moving these 2 lines (284 and 285) [ into the filer](https://github.com/bcgov/lear/pull/2903/files#diff-54617f2b54e8519023b9faaa7fbaf8083aaada150add761198d8972384613f40R105-R106), so the processing step will remain at `WARNING_LEVEL_2` in this job until the filing can be processed

Reply:
I was talking about keeping step change in the job but move status change to filer...not sure if that is what you want @argush3 💭

Reply:
> I was talking about keeping step change in the job but move status change to filer...not sure if that is what you want @argush3 💭

ok, sorry, i was getting confused a bit between step and status I think.

Yes, I think it makes sense to keep `batch_processing.step = BatchProcessing.BatchProcessingStep.DISSOLUTION` in the dissolutions job and to do the status change(`QUEUED` to `COMPLETED`) in filer.

@leodube-aot hopefully that makes sense.  I think doing it this way we'll be able to easily tell if a business got to stage 3 and if filer has processed the dissolution filing successfully yet.

Reply:
I moved the step process change back into the job, and made a note about the unit tests below. This PR is ready to have a second look at!